[All hosts](NAA) Reorg to recommend NAA SSO (#5570)
OfficeDev/office-js-docs-pr: docs/develop/authorize-to-microsoft-graph-without-sso.md

> [!NOTE]
> If you're implementing SSO and plan to access Microsoft Graph, see [Enable single sign-on in an Office Add-in with nested app authentication](enable-nested-app-authentication-in-your-add-in.md).

For information about programming authentication using the Microsoft identity platform, see [Microsoft identity platform documentation](/azure/active-directory/develop). You'll find tutorials and guides in that documentation set, as well as links to relevant samples. Once again, you may need to adjust the code in the samples to run in the Office dialog box to account for the Office dialog box that runs in a separate process from the task pane.

After your code obtains the access token to Microsoft Graph, it passes the access token from the dialog box to the task pane. (See [Authentication with the Office dialog API](auth-with-office-dialog-api.md) for details.) Code in the task pane requests data from Microsoft Graph and includes the token in those requests. For more information about calling Microsoft Graph and the Microsoft Graph SDKs, see [Microsoft Graph documentation](/graph/).

## Recommended libraries and samples

> We recommend that you use the following libraries when accessing Microsoft Graph.

Reviewer: ```suggestion
The following libraries provide access to Microsoft Graph.
```
